### ISSUE TITLE
$ sign in TEXT column

### ISSUE THREAD

**Issue by ar-jan** [pawelsalawa/sqlitestudio#3519]
### Details
I have a SQLite database which contains values like `$1` in a TEXT column. If I browse the table in data view I can see and edit the content, but if i do a `SELECT columnname FROM tablename`, the result shows an empty cell instead of the value. Similarly I can't filter on the value (`WHERE columnname LIKE '$%'`). Maybe I'm missing something, but I don't see anything about having to escape `$` in SQLite, or is there?

### Steps to reproduce
As above.

### Operating system
Windows 10 64-bit

### SQLiteStudio version
v3.2.1

**Comment by mrgchr** [comment]
In my local env(almost same as your OS, AppVer), I can't reproduce issues you mentioned.
`$` sign is shown in `select` result or filtered in `where` clause without any escaping.

![Data](https://user-images.githubusercontent.com/27002895/56402378-0af6ac80-6298-11e9-8b43-3dfa91908c21.png)

![Select](https://user-images.githubusercontent.com/27002895/56402382-0fbb6080-6298-11e9-883b-c5cecef5ef5c.png)

![Select2](https://user-images.githubusercontent.com/27002895/56402384-134ee780-6298-11e9-9033-a3c75e356f58.png)

```sql
CREATE TABLE DollarSample (
    Id    INTEGER PRIMARY KEY AUTOINCREMENT,
    Value TEXT
);
```

By chance, I'm guessing that your `$` sign in Data view is not an ASCII character '36(0x24)' , but is other similar character code. (just guessing, no confidence)
How about check chara code by copy value and paste from Data view (e.g. by using following web : **<http://www.mauvecloud.net/charsets/CharCodeFinder.html>**)

**Comment by ar-jan** [comment]
I created a new table like yours and saw `$` sign displayed in a select query as it should.

Then I looked at the database where I had this problem, and checked that it was indeed a `$` sign. But, somehow the exact same query which previously omitted values now works as expected! I have no idea how this is possible, it's the same table, unchanged (I did do modifications in other tables in the database). I guess I'll just close this.
